scripts/utils/our-resources-helpers.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * handleResponse
+ * 
+ * @param {object} response 
+ */
+export function handleResponse(response) {
+  if (response.status >= 400) {
+    throw new Error("Bad response from server");
+  }
+  return response.json();
+}
+
+/**
+ * responseToHTML
+ * 
+ * @param {object} param0 
+ */
+export function responseToHTML({ articles: posts }) {
+  let html = '';
+
+  posts.forEach((post) => {
+    const postHTML = createPost(post);
+    html += postHTML;
+  });
+
+  return html;
+}
+
+/**
+ * createPost
+ * 
+ * @param {object} post 
+ */
+export function createPost(post) {
+  // Extract properties
+  const {
+    title = "Dummy title",
+    description = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat, quod.",
+    url = "#",
+    images = {
+      desktop: "http://picsum.photos/407/267/?random",
+      tablet: "http://picsum.photos/643/267/?random",
+      mobile: "http://picsum.photos/270/184/?random"
+    }
+  } = post;
+
+  const html =
+`
+<div class="our-resources__posts__slides__slide">
```

Comment:
Probably it would have been more readable and maintainable to have these divs cloned and transformed from the HTML instead of creating them from scratch with JS, just in case you need to change their structure or classes.

Reply:
Something as WebComponents specification? (Although it is not yet supported by all browsers).

Reply:
No I was thinking in having that HTML code in the HTML index file and with JS clone the structure and/or fill the data inside the HTML nodes in a way that if you need to change CSS classes for example, you wouldn't need to update your JS file as well